domain/domain.py

```diff
@@ -144,7 +148,7 @@ def get_active_attributes(self):
             raise Exception("No attribute contains erroneous cells.")
         return set(itertools.chain(*result))
 
-    def get_corr_attributes(self, attr):
+    def get_corr_attributes(self, attr, thres):
         """
         get_corr_attributes returns attributes from self.correlations
         that are correlated with attr with magnitude at least self.cor_strength
```

Comment:
modify the docs according to the new `thres` parameter

Reply:
👍